compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -26,6 +26,8 @@ use crate::thir::cx::Cx;
 use crate::thir::util::UserAnnotatedTyHelpers;
 
 impl<'tcx> Cx<'tcx> {
+    // Create a THIR expression with the same structure as a HIR expression.
+    // (The term "mirror" in this case does not refer to "flipped" or "reversed".)
```

Comment:
```suggestion
    /// Create a THIR expression for the given HIR expression. This expands
    /// all adjustments and directly adds the type information from the
    /// `typeck_results`. See the [dev-guide] for more details.
    ///
    /// [dev-guide]: https://rustc-dev-guide.rust-lang.org/thir.html
```
Feel free to keep the explanation for "mirror" around if you think it's useful

Reply:
Thanks for the improved text. I added it now. Personally I think the "mirror" clarification is helpful so I kept it around.

Reply:
Maybe you expected me to click "Commit suggestion" so you would get attribution in the form of your own commit? I apologize if so. I can fix retroactively fix that if you want?

Reply:
:+1: would have been nice but also not something I necessarily mind :grin: so no worries